docs/docs/_index.md

```diff
@@ -9,7 +9,7 @@ type: docs
 [![Github](https://img.shields.io/static/v1?label=&message=repository&color=5961FF&logo=github)](https://github.com/RediSearch/RediSearch/)
 
 
-RediSearch is a [source-available](https://raw.githubusercontent.com/RediSearch/RediSearch/master/LICENSE.txt) Redis module that enables querying, secondary indexing, and full-text search for Redis. These features enable multi-field queries, [aggregation](/docs/stack/search/reference/aggregations), exact phrase matching, [numeric filtering](/docs/stack/search/reference/query_syntax/#numeric-filters-in-query), [geo filtering](/docs/stack/search/reference/query_syntax/#geo-filters-in-query) and [vector similarity semantic search](/docs/stack/search/reference/query_syntax/#vector-similarity-search-in-query) on top of text queries.
+Search and Query is a [source-available](https://raw.githubusercontent.com/RediSearch/RediSearch/master/LICENSE.txt) Redis module that enables querying, secondary indexing, and full-text search for Redis. These features enable multi-field queries, [aggregation](/docs/stack/search/reference/aggregations), exact phrase matching, [numeric filtering](/docs/stack/search/reference/query_syntax/#numeric-filters-in-query), [geo filtering](/docs/stack/search/reference/query_syntax/#geo-filters-in-query) and [vector similarity semantic search](/docs/stack/search/reference/query_syntax/#vector-similarity-search-in-query) on top of text queries.
```

Comment:
```suggestion
Search and Query is part of the enhanced Redis experience with Redis Stack that provides querying capabilities, secondary indexing, full-text search, vector similarity search, and more. To use Search and Query, you first declare indices on your Redis data and then use the query language to retrieve the data asset that your application requires. All future data will be also be indexed automatically.
```

Reply:
@adrianoamaral is this version ok?

Reply:
Did some changes to add more key features and aiming for a single Redis experience

Reply:
"Redis experience with Redis" seems redundant. May I change that to simply "Redis experience that provides..."?